datatable_tutorial/datatable_tutorial/datatable_tutorial.py

```diff
@@ -0,0 +1,269 @@
+import reflex as rx
+from typing import Any
+import asyncio
+import httpx
+
+
+class BaseState(rx.State):
+    pass
+
+   
+class DataTableState(BaseState):
+    """The app state."""
+
+    clicked_cell: str = "Cell clicked: "
+    edited_cell: str = "Cell edited: "
+    right_clicked_group_header : str = "Group header right clicked: "
+    item_hovered: str = "Item Hovered: "
+    deleted: str = "Deleted: "
+
+    cols: list[dict] = [
+        {"title": "Title", "type": "str"},
+        {
+            "title": "Name",
+            "type": "str",
+            "group": "Data",
+            "width": 300,
+        },
+        {
+            "title": "Birth",
+            "type": "str",
+            "group": "Data",
+            "width": 150,
+        },
+        {
+            "title": "Human",
+            "type": "bool",
+            "group": "Data",
+            "width": 80,
+        },
+        {
+            "title": "House",
+            "type": "str",
+            "group": "Data",
+        },
+        {
+            "title": "Wand",
+            "type": "str",
+            "group": "Data",
+            "width": 250,
+        },
+        {
+            "title": "Patronus",
+            "type": "str",
+            "group": "Data",
+        },
+        {
+            "title": "Blood status",
+            "type": "str",
+            "group": "Data",
+            "width": 200,
+        },
+    ]
+
+    data = [
```

Comment:
```suggestion
    data: list[str] = [
```

should still provide an annotation

Reply:
fixed